Add Shell scripts (#40)
signalfx/splunk-otel-dotnet: docs/README.md

> Some settings can be omitted on .NET (Core). For more information, see the [documentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.3.1-beta.1/docs/config.md#net-clr-profiler).

### Shell scripts

> You can install Splunk Distribution of OpenTelemetry .NET
> and instrument your .NET application using the provided Shell scripts.
> Example usage:

Reviewer: ```suggestion
You can install the Splunk Distribution of OpenTelemetry .NET
and instrument your .NET application using the official shell scripts.

For example:
```

Reviewer: Can we explain the difference between the install and instrument scripts?

Author: How about:

```
The [splunk-otel-dotnet-install.sh](../splunk-otel-dotnet-install.sh) script
installs the OpenTelemetry .NET Automatic Instrumentation.
It uses environment variables as parameters:
```

```
The [instrument.sh](../instrument.sh) script
enables the instrumentation in the current shell session.
It uses environment variables as parameters:
```

I am not sure how to make it clear that "install" only "places the component on local machine".

Reviewer: So instrument includes install ? Or they're just separate steps to be run in sequence? If that's the case, what's the reason for splitting them?

Author: Will be addressed in a separate PR

Author: https://github.com/signalfx/splunk-otel-dotnet/pull/42

```sh
curl -sSfL https://raw.githubusercontent.com/signalfx/splunk-otel-dotnet/v0.0.1-alpha.1/splunk-otel-dotnet-install.sh -O
sh ./splunk-otel-dotnet-install.sh
. $HOME/.splunk-otel-dotnet/instrument.sh
OTEL_SERVICE_NAME=myapp OTEL_RESOURCE_ATTRIBUTES=deployment.environment=staging,service.version=1.0.0 dotnet run
```

> [splunk-otel-dotnet-install.sh](../splunk-otel-dotnet-install.sh) script

Reviewer: ```suggestion
The [splunk-otel-dotnet-install.sh](../splunk-otel-dotnet-install.sh) script
```

uses environment variables as parameters:

| Parameter               | Description                                                      | Required | Default value             |
|-------------------------|------------------------------------------------------------------|----------|---------------------------|
| `OTEL_DOTNET_AUTO_HOME` | Location where binaries are to be installed                      | No       | `$HOME/.splunk-otel-dotnet` |
| `OS_TYPE`               | Possible values: `linux-glibc`, `linux-musl`, `macos`, `windows` | No       | *Calculated*              |
| `TMPDIR`                | Temporary directory used when downloading the files              | No       | `$(mktemp -d)`            |
| `VERSION`               | Version to download                                              | No       | `v0.0.1-alpha.1`           |

> [instrument.sh](../instrument.sh) script

Reviewer: ```suggestion
The [instrument.sh](../instrument.sh) script
```
